test/html/expect/test_package+ml/Nested/F/argument-1-Arg1/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml">
  <head>
   <title>
-   1-Arg1 (test_package+ml.Nested.F.1-Arg1)
+   Arg1 (test_package+ml.Nested.F.1-Arg1)
```

Comment:
Ok, a nit. Or perhaps a question to be addressed in a future issue: should the path in the parentheses also end in `Arg1`, rather than `1-Arg1`?

Reply:
I ... felt it was better that way. The path, on the other hand should preferably be unique and can be richer/clunkier. It's certainly not difficult to change, one way or the other.